gateway/server.go

```diff
@@ -1196,6 +1196,14 @@ func getGlobalStorageHandler(keyPrefix string, hashKeys bool) storage.Handler {
 	return &storage.RedisCluster{KeyPrefix: keyPrefix, HashKeys: hashKeys}
 }
 
+func NewGateway() Gateway {
```

Comment:
I think it would be better if we receive a config here and maintain that config in our Gateway struct.

Reply:
done